src/main/java/org/takes/rs/RsVelocity.java

```diff
@@ -131,7 +132,7 @@ public InputStream body() throws IOException {
      * @throws IOException If fails
      */
     private static InputStream render(final InputStream page,
-        final Map<String, Object> params) throws IOException {
+        final Map<CharSequence, Object> params) throws IOException {
```

Comment:
@carlosmiranda Why do I need to make code complicated, where it is not necessary? Now we  can  render with params as "new HashMap<String,Object>()" and it will works. @yegor256 help please